schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -211,14 +211,21 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-05-01-preview/Microsoft.MachineLearningExperimentation.json#/resourceDefinitions/accounts_workspaces"},
                   { "$ref": "https://schema.management.azure.com/schemas/2017-05-01-preview/Microsoft.MachineLearningExperimentation.json#/resourceDefinitions/accounts_workspaces_projects"},
                   { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/automationAccounts" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/runbooks" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/modules" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/certificates" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/connections" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/variables" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/schedules" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/jobs" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/jobSchedules" },
```

Comment:
is this going to break existing templates?

Reply:
The earlier schema was hand-written and old (2 years ago). I generated the new ones from Autorest and put renamed them here. The resources themselves haven't changed, but I can double check with my team to make sure that this deploymentTemplate does not break existing templates.

Reply:
Hi @TianoMS,

I tested with existing templates as well as a custom template I made and based on my testing, the changes are backward compatible.

```
varads-mbp:tools varadmeru$ npm test

> template-schema-validation-tools@1.0.0 test /Users/varadmeru/work/microsoft/github/azure-resource-manager-schemas/tools
> mocha runSchemaValidation.js

  validate template files with schema - 
    ✓ running schema validation on 'Template2016.json' (665ms)
    ✓ running schema validation on 'Template2016.preview.json' (265ms)
    ✓ running schema validation on 'TestTemplate.json' (1020ms)
    ✓ running schema validation on 'TestTemplate.preview.json' (807ms)
    ✓ running schema validation on 'Various.json' (289ms)

  5 passing (3s)

```